Simplenote/NoteListViewController.m

```diff
@@ -84,11 +84,7 @@ - (void)viewDidLoad
     [self setupTableView];
     [self startListeningToScrollNotifications];
     [self startListeningToWindowNotifications];
-}
 
-- (void)viewWillAppear
-{
-    [super viewWillAppear];
```

Comment:
I'm not entirely sure how this change slipped in here (It's something I've been working on in another branch!).

Please disregard this one, **should** hopefully be inoccuous!!